patches/lasindex.cpp

```diff
@@ -0,0 +1,48 @@
+#include <sciplot/sciplot.hpp>
```

Comment:
Hmm, where did this file come from? Is this copied from upstream? Is there a way to call this or harness separately? As it is this will likely diverge from upstream at some point.